afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/RemoteAppStorage.java

```diff
@@ -998,9 +1003,10 @@ public boolean isClosed() {
     }
 
     @Override
-    public void close() {
+    public synchronized void close() {
```

Comment:
Not really sure that it's OK: during closing, what happens if the flush() method is called?

Reply:
It was a mistake to add this, it has nothing to do with the feature, and it seems reasonable to assume only 1 thread will try to close this.
I will remove it.

About the question : the flush method of the buffer seems to use a lock, so there should not be concurrent executions of this.

Reply:
Indeed, I didn't notice that the changeBuffer was thread-safe.